src/main/java/games/strategy/engine/framework/startup/ui/PBEMSetupPanel.java

```diff
@@ -431,15 +431,6 @@ public void propertyChange(final PropertyChangeEvent evt) {
     }
   }
 
-  public String getPlayerType(final String playerName) {
```

Comment:
This method was introduced in 8582401, but there appears to never have been a call to it since.